### ISSUE TITLE
Too many logins

### ISSUE THREAD

**Issue by Pheerink** [ralfvd/nl.bevlogenheid.evohome#16]
Since a few days I had problems logging in at Honeywell (too many logins). My account was blocked due to this. It is a crime to get unblocked again via Honeywell but today they gave me access again. After I reseted my account in Homey and I got a huge log with successful login attempts.  But after a few minutes the Honeywell account was blocked again due to "too many logins". What's going on with the logins ? Why is it loggin in so often? Is there something wrong ?

**Comment by psy0rz** [comment]
Duplicate of #15
